app/src/main/java/fr/free/nrw/commons/review/ReviewController.java

```diff
@@ -135,24 +141,56 @@ public void reportWrongCategory(@NonNull Activity activity) {
                 }, Timber::e);
     }
 
-    private void publishProgress(@NonNull Context context, int i) {
-        int[] messages = new int[]{R.string.getting_edit_token, R.string.check_category_adding_template};
+    /**
+     * @param context
+     * @param i progress as an integer
+    * While reportWrongCategory() is in background notify the user about current progress in the newtwork request*/
+
+    private void publishProgressForWrongCategory(@NonNull Context context, int i) {
+        int[] listOfMessages = new int[]{R.string.getting_edit_token, R.string.check_category_adding_template};
```

Comment:
What I meant was, this is not listOfMessages. This is list of integers which are ids of messages.

Reply:
I am sorry, I have misunderstood that before :(. I have made the changes in the latest commit.